### PR TITLE
The First Push aka A ReadMe

### PR DESCRIPTION
**Pull request by EricqyHan**
Created A Read Me. Feel free to edit it and make changes.

### CODE REVIEW

**Comment by dotRollen** [comment]
👍  Looks good but let's keep the references of the bootcamp out. The second portion `ELI5: For noncoders` should be the main introduction.

**Comment by dotRollen** [comment]
Hey @EricqyHan  I don't see the changes made to the readme

**Comment by cimafrancad45** [comment]
Looks pretty good for the readme. I changed a few things in the readme.

**Comment by Nikee-R** [comment]
I made a couple changes to the flow(?) of the ReadMe. I was trying to tie together the ideas nicely lol. I think it is good!
